python/lsst/afw/table/source.cc

```diff
@@ -112,6 +112,8 @@ void declareSourceRecord(PySourceRecord & cls) {
     cls.def("updateCoord",
             (void (SourceRecord::*)(image::Wcs const &, PointKey<double> const &)) &SourceRecord::updateCoord,
             "wcs"_a, "key"_a);
+
+    cls.def_property_readonly("table", &SourceRecord::getTable);
```

Comment:
I think `baseColumnView.cc` already has this property. Sorry. But yes, all record types should have `getTable` and `table` as a property, preferably defined adjacent to each other (as in `baseColumnView.cc` so that it's trivial to see that both exist.